### PR TITLE
ENH put a dash in the name

### PR DESCRIPTION
**Pull request by beckermr**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by beckermr** [comment]
@conda-forge-admin rerender

**Comment by cwwalter** [comment]
If this will be used as a base environment with additional possible subsidiary metapackages (as suggested by @ktlim in slack), then I suggest that `rubin-base-env` might be a good name.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
-{% set name = "rubinenv" %}
+{% set name = "rubin-env" %}
```
Comment (beckermr) [inline recipe/meta.yaml:1]:
```suggestion
{% set name = "rubin-base-env" %}
```

**Comment by beckermr** [comment]
@conda-forge-admin rerender

**Comment by timj** [comment]
I'm fine with rubin-base-env -- it is clearer. I guess @ktlim and @brianv0 need to weigh in.

**Comment by brianv0** [comment]
I had a goal to have rubin-env as the touchstone recipe with multiple outputs for sims, user analysis, t&s - ideally with variants you could just install, so I'd slightly prefer to keep it rubin-env, but rubin-base-env could be better in the near term.

**Comment by beckermr** [comment]
Any maintainer of this feedstock can edit my fork, so pls put in w/e name is best for you all and merge!

Make sure to rerender before doing so like I did above.

**Comment by cwwalter** [comment]
Variants would be nice.  One possible overloading that just occurred to me: rubin-base-env is simple and clear.  "base" does have another meaning for us though that is used in some other contexts.

**Comment by ktlim** [comment]
Chris's point is good that "base" is an overloaded term.  And if additional variants are not layered on top as their own separate feedstocks, then I think `rubin-env` is appropriately inclusive.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
-{% set name = "rubinenv" %}
+{% set name = "rubin-base-env" %}
```
Comment (beckermr) [inline recipe/meta.yaml:1]:
```suggestion
{% set name = "rubin-env" %}
```

**Comment by beckermr** [comment]
@conda-forge-admin rerender

**Comment by brianv0** [comment]
Does this repo need to be renamed to reflect these changes?

**Comment by beckermr** [comment]
Nope. The reponame is separate from the package.

**Pull request merged**